packages/components/src/index.js

```diff
@@ -14,7 +14,7 @@ export {
 export { default as __experimentalAlignmentMatrixControl } from './alignment-matrix-control';
 export {
 	default as Animate,
-	useAnimate as __unstableUseAnimate,
+	getAnimateClassName as __experimentalGetAnimateClassName,
```

Comment:
I just noticed this was `__unstable` - not intended for use outside of Gutenberg. Should it stay that way?

Reply:
If we're confident in the API, we can stabilise it? The question is, do we intend to use this for future animations, or are there new methods there days? Cc @jasmussen.

Reply:
I'm sad to say I've not had enough of a chance to leverage this one quite yet, to make a call. However I recall @youknowriad being pretty excited about it?

Reply:
Maybe I can land this PR with the rename and keep it marked unstable. A follow-up could mark it experimental or stabilize.